playbooks/roles/xapi_bridge/tasks/main.yml

```diff
@@ -60,3 +60,63 @@
     state: present
   become_user: "{{ xapi_bridge_user }}"
   tags: ["xapi_bridge"]
+
+- name: Create symlinks from the venv bin dir and repo dir
+  file:
+    src: "{{ item }}"
+    dest: "{{ COMMON_BIN_DIR }}/{{ item.split('.')[0] }}.xapi_bridge"
+    state: link
+  with_items:
+    - '{{ xapi_bridge_venv_bin }}/python'
+  tags: ["xapi_bridge"]
+
+- name: create the xapi log directory
+  file:
+    path: "{{ COMMON_LOG_DIR }}/{{ xapi_bridge_user }}"
+    state: directory
+    mode: 0755
+    owner: "{{ xapi_bridge_log_user }}"
+    group: "{{ xapi_bridge_log_user }}"
+
+- name: create the log file
+  file:
+    path: "{{ COMMON_LOG_DIR }}/{{ xapi_bridge_user }}/{{ item }}"
+    state: touch
+    mode: 0644
+    owner: "{{ xapi_bridge_log_user }}"
+    group: "{{ xapi_bridge_log_user }}"
+  with_items:
+    - 'xapi_stderr.log'
+    - 'xapi_stdout.log'
```

Comment:
In the python application, I have set logging to log everything to a single file, at level INFO 

```py
    if getattr(settings, 'DEBUG_MODE', False):
        logging.basicConfig(
            format='%(levelname)s:%(message)s',
            level=logging.DEBUG
        )
    else:
        logging.basicConfig(
        filename='/edx/var/log/xapi/xapi_bridge.log',
        filemode='a+',
        format='%(levelname)s:%(message)s',
        level=logging.INFO
)
```

This doesn't match your stderr + stdout capture setup here.  

What do you think we should do?  The non-DEBUG setup here will send all logged messages, info, warnings, handled errors to a single file.  It's possible to filter errors out to stderr I think but I think it's more work than necessary.

Reply:
@bryanlandia yeah. I think what you're doing is correct. I wasn't aware of this. Let's wait for @tkeemon and see what he thinks about this.

Reply:
Once again, system-level logging and application level logging are kept separate so it's fine what you're doing.